base/applications/charmap/CMakeLists.txt

```diff
@@ -10,6 +10,6 @@ list(APPEND SOURCE
 add_rc_deps(charmap.rc ${CMAKE_CURRENT_SOURCE_DIR}/res/charmap.ico)
 add_executable(charmap ${SOURCE} charmap.rc)
 set_module_type(charmap win32gui UNICODE)
-add_importlibs(charmap advapi32 user32 gdi32 comctl32 msvcrt kernel32)
+add_importlibs(charmap advapi32 user32 gdi32 comctl32 msvcrt kernel32 getuname)
```

Comment:
New exports go to the left, msvcrt/kernel32/ntdll must the the last in the list (if present).